### PR TITLE
Consistency Fixes

### PR DESCRIPTION
**Pull request by SteveLasker**
A set of consistency fixes for the next draft.

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ The rise in popularity of verifiable data structures as a mechanism to make acto
 
 This document defines a generic, interoperable and scalable architecture to enable transparency across any supply chain with minimum adoption barriers.
 It provides flexibility, enabling interoperability across different implementations of Transparency Services with various auditing and compliance requirements.
-Issuers can register their Signed Statements on any Transparency Service, with the guarantee that all Consumers will be able to verify them.
+Issuers can register their Signed Statements on any Transparency Service, with the guarantee that all Consumers[TODO: Fix Resolve Consumer] will be able to verify them.
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:145]: I'm flagging the term "Consumer" as we previously had Producers and Consumers. Tagging for an alternative term.

Comment (aj-stein-nist) [inline draft-ietf-scitt-architecture.md:145]: I would recommend this or I can drive alignment after these fixes are merged and continue the work in #120.

```suggestion
Issuers can register their Signed Statements on any Transparency Service, with the guarantee that all Verifiers and Relying Parties will be able to verify them.
```

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -179,10 +179,8 @@ Considering CT in terms of SCITT:
 - CAs (Issuers) sign X.509 TBSCertificates (Artifacts) to produce X.509 certificates (Signed Statements)
 - CAs submit the certificates to one or more CT logs (Transparency Services)
 - CT logs produce Signed Certificate Timestamps (Transparent Statements)
-- SCTs are checked by browsers (Verifiers)
-- The Append-only Log can be checked by Auditors
-
-Note that just like CT logs are independent and their contents need not be consistent, Transparency Services are similarly independent of each other.
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:185]: This sentence is poorly worded. Open to a suggestion, if needed

**Review by OR13 — DISMISSED** [review]
I would start with the new roles that are needed... lets not add a bunch of TODOs.

define the new roles you need, and then use them consistently throughout the document.

**Comment by SteveLasker** [comment]
See https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/180 for discussion on the terminology.

File: draft-ietf-scitt-architecture.md (hunk 1 above)
Comment (SteveLasker) [reply draft-ietf-scitt-architecture.md]: Thanks, @aj-stein-nist 
I reduced this PR to be more about consistency, and suggest we do a separate PR to resolve using Consumer or Relying Parties. I do think we can do something better, but would like to track that one separately.

File: draft-ietf-scitt-architecture.md [hunk 3]
```diff
@@ -179,10 +179,8 @@ Considering CT in terms of SCITT:
...
+- Signed Certificate Timestamps are checked by browsers (Verifiers)
```
Comment (aj-stein-nist) [inline draft-ietf-scitt-architecture.md:182]: Not all verifiers will be "[web] browsers" per se.

```suggestion
- Signed Certificate Timestamps are checked by Verifiers.
```

**Review by aj-stein-nist — APPROVED** [review]
It cuts out cruft, so I support this edit, just one minor nit (take or leave it, it's minor to me), before moving onto #179.

File: draft-ietf-scitt-architecture.md (hunk 3 above)
Comment (SteveLasker) [reply draft-ietf-scitt-architecture.md]: Great clarification

**Pull request merged**